[2022-07-22 13:00:19,734] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-23T00:00:00+00:00 [queued]>
[2022-07-22 13:00:19,819] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-23T00:00:00+00:00 [queued]>
[2022-07-22 13:00:19,829] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 13:00:19,842] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-22 13:00:19,843] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 13:00:19,967] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-06-23 00:00:00+00:00
[2022-07-22 13:00:20,002] {standard_task_runner.py:52} INFO - Started process 810 to run task
[2022-07-22 13:00:20,027] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-06-23T00:00:00+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpvclzrw6y', '--error-file', '/tmp/tmpded9z7yn']
[2022-07-22 13:00:20,033] {standard_task_runner.py:80} INFO - Job 52: Subtask t2
[2022-07-22 13:00:20,154] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-22 13:00:20,462] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-06-23T00:00:00+00:00 [running]> on host 2f63307804d5
[2022-07-22 13:00:20,848] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-06-23T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-23T00:00:00+00:00
[2022-07-22 13:00:20,853] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-22 13:00:20,991] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220623T000000, start_date=20220722T130019, end_date=20220722T130020
[2022-07-22 13:00:21,098] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-22 13:00:21,223] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-25 09:18:04,240] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-23T00:00:00+00:00 [queued]>
[2022-07-25 09:18:04,301] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-23T00:00:00+00:00 [queued]>
[2022-07-25 09:18:04,302] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:18:04,303] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-25 09:18:04,304] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:18:04,368] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-06-23 00:00:00+00:00
[2022-07-25 09:18:04,394] {standard_task_runner.py:52} INFO - Started process 770 to run task
[2022-07-25 09:18:04,407] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-06-23T00:00:00+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmp5sveyw6p', '--error-file', '/tmp/tmpp05463uk']
[2022-07-25 09:18:04,413] {standard_task_runner.py:80} INFO - Job 51: Subtask t2
[2022-07-25 09:18:04,492] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-25 09:18:04,780] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-06-23T00:00:00+00:00 [running]> on host 01e6b1df1873
[2022-07-25 09:18:05,051] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-06-23T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-23T00:00:00+00:00
[2022-07-25 09:18:05,075] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-25 09:18:05,300] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220623T000000, start_date=20220725T091804, end_date=20220725T091805
[2022-07-25 09:18:05,393] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-25 09:18:05,595] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-25 12:26:56,600] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-23T00:00:00+00:00 [queued]>
[2022-07-25 12:26:56,695] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-23T00:00:00+00:00 [queued]>
[2022-07-25 12:26:56,701] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 12:26:56,704] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-25 12:26:56,705] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 12:26:56,840] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-06-23 00:00:00+00:00
[2022-07-25 12:26:56,880] {standard_task_runner.py:52} INFO - Started process 844 to run task
[2022-07-25 12:26:56,933] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-06-23T00:00:00+00:00', '--job-id', '120', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmphq_janm3', '--error-file', '/tmp/tmpcmvsfioh']
[2022-07-25 12:26:56,945] {standard_task_runner.py:80} INFO - Job 120: Subtask t2
[2022-07-25 12:26:57,089] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-25 12:26:57,427] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-06-23T00:00:00+00:00 [running]> on host 929e6a3728d4
[2022-07-25 12:26:57,800] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-06-23T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-23T00:00:00+00:00
[2022-07-25 12:26:57,808] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-25 12:26:58,260] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220623T000000, start_date=20220725T122656, end_date=20220725T122658
[2022-07-25 12:26:58,368] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-25 12:26:58,581] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
